cogs/socials.py

```diff
@@ -234,11 +235,13 @@ async def gay(self, ctx, user=None, border=False, server_avatar=True):
     @option("text", str, description="What do you want to tell Paw?")
     async def gpt(self, ctx: discord.ApplicationContext, text: str):
         """ Talk to Paw! """
+        if not self.ai_enabled:
+            return await ctx.respond("AI functions are disabled due to missing (or invalid) API key, please contact the bot owner to fix this.")
```

Comment:
Make the response ephemeral, just say "AI functions are currently disabled" without the last part and log it instead (look at how I log stuff elsewhere in the bot)